buildenv/jenkins/variables/defaults.yml

```diff
@@ -407,8 +407,8 @@ x86-64_linux:
       8: 'ci.role.build && hw.arch.x86 && sw.tool.docker'
       11: 'ci.role.build && hw.arch.x86 && sw.tool.docker'
     docker_image:
-      8: 'eclipseopenj9/jenkins-agent-x86-centos6:15'
-      11: 'eclipseopenj9/jenkins-agent-x86-centos6:15'
+      8: 'eclipseopenj9/jenkins-agent-x86-centos6:16'
+      11: 'eclipseopenj9/jenkins-agent-x86-centos6:16'
```

Comment:
Why do we not want to use `latest` instead of `16`?

Reply:
This is our service mindset coming though. If we need to rebuild something old, the release branch will contain the right version of the container that was used at the time.

Reply:
Just a guess, but using `latest` means we could be broken unexpectedly.

Reply:
Got it. Thanks.